swap_meet/item.py

```diff
@@ -14,18 +14,32 @@ def __str__(self):
         return f"An object of type {self.get_category()} with id {self.id}."
         
     def condition_description(self):
-        if self.condition == 0:
-            return "Trash"
-        if self.condition == 1:
-            return "Heavily used"
-        if self.condition == 2:
-            return "Between the rag box and donation"
-        if self.condition == 3:
-            return "Moderately used"
-        if self.condition == 4:
-            return "Lightly used"
         if self.condition == 5:
             return "Like new"
+        if self.condition >= 4:
+            return "Lightly used"
+        if self.condition >= 3:
+            return "Moderately used"
+        if self.condition >= 2:
+            return "Between the rag box and donation"
+        if self.condition >= 1:
+            return "Heavily used"
+        if self.condition >= 0:
+            return "Trash"
```

Comment:
👍 This approach accounts for the gaps between the fixed condition values.

Of the two updated approaches, I prefer this one, since it doesn't require naming both ends of the conditions (which is repetitive). Using the single end approach also _forces_ us to write the conditions in a logical order.

Note that both approaches still are using chained conditions, which are brittle no matter the condition style. I would still sugget thinking about how the condition range information could be stored in data structure (e.g., list or dict), and use that determine the description to pick. Once we have a data structure, and a dependable way to lookup values in it, it would be easier to update the descriptions in a safe fashion.